includes/Feature.php

```diff
@@ -44,22 +44,22 @@ final public function __construct( $options ) {
 		$this->options = $options;
 
 		// check if state already saved to options
-		$savedAs = $this->getOption();
+		$savedAs = $this->options->get( $this->name );
```

Comment:
```suggestion
		$this->value = $this->options->get( $this->name );
```
Since the default return value is `false` unless it is set.